janus_core/helpers/utils.py

```diff
@@ -3,6 +3,42 @@
 from pathlib import Path
 from typing import Optional
 
+from ase import Atoms
+from spglib import get_spacegroup
+
+
+def spacegroup(
+    struct: Atoms, sym_precision: float = 0.001, angle_precision: float = -1.0
+) -> str:
+    """
+    Determine the spacegroup for a structure.
+
+    Parameters
+    ----------
+    struct : Atoms
+        Structure as an ase Atoms object.
+    sym_precision : float
+        Symmetry precision for spglib symmetry determination.
```

Comment:
Seems it should be described in length or max deviation, max offset, since the other is referred to as "angle"

Reply:
Having thought on it for a bit `tolerance` would be better than precision for these variables

Reply:
done